tutorials/advanced_features/Simulating Sequences with Errors and Noise.ipynb

```diff
@@ -1,36 +1,49 @@
 {
```

Comment:
<ul><li>"it has two child class<strong>es</strong>, <strong> </strong>the <code>CoherentResults</code> for <strong>when the final state still can be represented as a state vector or density matrix and </strong><code><strong>NoisyResults</strong></code><strong> when it can only be provided as a probability distribution of final states.</strong>"  </li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/460/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>